experiment/measurer/run_crashes.py

```diff
@@ -30,14 +30,21 @@
     'crash_testcase', 'crash_type', 'crash_address', 'crash_state',
     'crash_stacktrace'
 ])
+
 SIZE_REGEX = re.compile(r'\s([0-9]+|{\*})$', re.DOTALL)
+TEMPLATE_REGEX = re.compile(r'<[^>]+>')
```

Comment:
Could you make it explicit that this is C++ templates?
I initial thought this had something to do with HTML templates.

Reply:
Done!